static/tile.js

```diff
@@ -15,13 +15,27 @@ function createTile(element) {
   var temp = document.createElement('div');
   temp.className = "grid-item"
   // Format tile
-  temp.innerHTML = '<div class="card"><div class="card-body"><h5 class="card-title">' +
+  try {
+    temp.innerHTML = '<div class="card"><div class="card-body"><h5 class="card-title">' +
                     element._metadata.tile + 
-                    '</h5><p class="card-text">' + 
+                    '</h5>' +
+                    '<img src="' + element.image +'" alt="IM Image">' +
+                    '<p class="card-text">' + 
                     unpackContent(element) + 
                     '</p><p class="card-text"><small class="text-muted">Created by ' +
                     element._metadata.creator +
                     '</small></p></div></div>';
+  } catch(error) {
+    temp.innerHTML = '<div class="card"><div class="card-body"><h5 class="card-title">' +
+                    element._metadata.tile + 
+                    '</h5>' +
+                    '<p class="card-text">' + 
+                    unpackContent(element) + 
+                    '</p><p class="card-text"><small class="text-muted">Created by ' +
+                    element._metadata.creator +
+                    '</small></p></div></div>';
+  }
+  
```

Comment:
This code assumes everything is an image, hopes that a non-image throws an exception, and then delivers the default behavior on exception.  The image is the rare case and should be the special path -- the IMs may need to prefix the image with an identifier (ex: the key `data:image/png;base64` is the standard base64 image identification string).

Reply:
Yeah I did do that in a hacky way.

![image](https://user-images.githubusercontent.com/5403037/145644584-659fe43f-18d0-4ce5-969c-d5ab8e8ed232.png)

I initially tried something like this thinking it would work. However, when displaying it shows the concatenated information outside of the div for some reason why boggles me.
![image](https://user-images.githubusercontent.com/5403037/145644678-ca8297cf-f202-41aa-8437-f051906aee85.png)

Reply:
I just ended up removing the image feature